### PR TITLE
refactor(Telephony- Group-number): Group-number page alignment

### PR DESCRIPTION
**Pull request by feoche**
## Title of the Pull Requests
Group-number page alignment

### Description of the Change
Group-number page alignment with oui-kit

### CODE REVIEW

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 1]
```diff
@@ -257,7 +257,7 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
                             </span>
                             <div data-ng-if="$ctrl.loading.webAccess">
                                 <span data-translate="telephony_number_feature_conference_general_settings_public_address_generating"></span>
-                                <spinner class="left-space-m4"></spinner>
+                                <bouncing-box-loader class="ml-2"></bouncing-box-loader>
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: missing `data-size` attribut?

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 2]
```diff
@@ -278,7 +278,7 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
...
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: missing `data-size` attribut?

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 3]
```diff
@@ -333,18 +333,18 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
...
                                accept="audio/*"
                                data-ng-model="$ctrl.model.announceUploadedFile"
                                data-input-file-change="$ctrl.onSoundFileChoosed" />
-                        <div class="dtable">
-                            <div class="dtd vertical-middle left-space-p16 right-space-p16">
+                        <div class="d-table">
+                            <div class="d-table-cell align-middle px-3">
                                 <i class="ovh-font ovh-font-upload fs18" data-ng-if="!$ctrl.loading.announceUpload"></i>
-                                <spinner data-ng-if="$ctrl.loading.announceUpload"></spinner>
+                                <bouncing-box-loader data-ng-if="$ctrl.loading.announceUpload"></bouncing-box-loader>
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: missing `data-size` attribut?

File: client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html [hunk 4]
```diff
@@ -19,67 +19,61 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_dialplan_edit"></h2>
                     <!-- NAME -->
                     <div class="section-item form-group"
                          data-ng-class="{ 'has-error': dialplanForm.name.$invalid && dialplanForm.name.$dirty }">
```
Comment (frenautvh) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html]: $touched

File: client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-edit.html [hunk 5]
```diff
@@ -161,49 +158,41 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_step_advanced"></h2>
...
-                                           data-ng-model="$ctrl.extension.schedulerCategory"
-                                           data-ng-value="null"
-                                           data-ng-change="$ctrl.parentCtrl.popoverStatus.move = false" />
-                                </flat-radio>
-                            </div>
-                            <div class="dtd vertical-middle">
-                                <span data-translate="telephony_number_feature_ovh_pabx_step_no_conditions"></span>
-                            </div>
+                    <label class="section-item item-choice w-100 d-table">
+                        <div class="d-table-cell vertical-middle radio-content">
```
Comment (frenautvh) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-edit.html]: align-middle

File: client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-edit.html [hunk 6]
```diff
@@ -161,49 +158,41 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_step_advanced"></h2>
...
+                            <input type="radio"
+                                   name="eventCategories"
+                                   data-ng-model="$ctrl.extension.schedulerCategory"
+                                   data-ng-value="null"
+                                   data-ng-change="$ctrl.parentCtrl.popoverStatus.move = false" />
+                        </div>
+                        <div class="d-table-cell vertical-middle">
```
Comment (frenautvh) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-edit.html]: align-middle

File: client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-edit.html [hunk 7]
```diff
@@ -161,49 +158,41 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_step_advanced"></h2>
...
-                                           data-ng-value="category"
-                                           data-ng-change="$ctrl.parentCtrl.popoverStatus.move = false" />
-                                </flat-radio>
-                            </div>
-                            <div class="dtd vertical-middle">
-                                <voip-time-condition-slot data-time-condition-slot="$ctrl.convertCategoryToSlot(category)"
-                                                          data-is-scheduler="true">
-                                </voip-time-condition-slot>
-                            </div>
+                        <div class="d-table-cell vertical-middle radio-content">
```
Comment (frenautvh) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-edit.html]: grep "vertical-middle" ;)

**Comment by feoche** [comment]
Closed bc @GenghisJean will work on this branch

**Pull request closed without merging**